backend/seeder.js

```diff
@@ -0,0 +1,75 @@
+require("dotenv").config({ path: __dirname + "/./.env" })
+const mongoose = require("mongoose")
+const axios = require("axios")
+const Character = require("./models/character.model")
+const Episode = require("./models/episode.model")
+const database_url = process.env.DATABASE_URL
+
+seed = async () => {
+  connectDb()
+  await seedCharacters()
+  await seedEpisodes()
+  closeDb()
+}
+
+connectDb = async () => {
+  mongoose
+    .connect(database_url, {
+      useNewUrlParser: true,
+      useUnifiedTopology: true,
+    })
+    .then(() => {
+      console.log("Database Connected")
+      return true
+    })
+    .catch((err) => {
+      console.error("Error : Cannot connect to Database " + err)
+      return false
+    })
+}
+
+closeDb = async () => {
+  mongoose.connection.close()
+}
+
+seedCharacters = async () => {
+  let charactersEndpoint = "https://finalspaceapi.com/api/v0/character"
```

Comment:
what about to store the BASE_URL_API=https://finalspaceapi.com/api on the .env.example
```suggestion
  let charactersEndpoint = "${BASE_URL_API}/v0/character"
```

Reply:
I guess we need another environment variable  something related to PORT for this.
Create a new environment variable and use it instead here.

Reply:
that is which I wanted to say, hahaha